lib/galaxy/tools/parameters/basic.py

```diff
@@ -1657,6 +1657,8 @@ def _get_history( self, trans, history=None ):
         assert trans is not None, "%s requires a trans" % class_name
         if history is None:
             history = trans.get_history()
+        if history is None:
+            history = trans.get_history( create=True )
```

Comment:
Instead of a second set of lines, should we just pass create=True to the invocation above?